.gitlab-ci.yml

```diff
@@ -41,7 +41,7 @@ sniffers:compose:
 
 sniffers:front:
   cache:
-    key: "front-packages"
+    key: "$CI_COMMIT_REF_NAME"
```

Comment:
I think we should better use `$CI_COMMIT_REF_NAME:$CI_COMMIT_SHA` as in https://github.com/skilld-labs/skilld-docker-container/blob/master/.gitlab-ci.yml#L116

Reply:
Or mb better use `${CI_COMMIT_REF_SLUG}` as suggested in docs https://docs.gitlab.com/ee/ci/caching/#sharing-caches-across-the-same-branch

Reply:
After reading https://docs.gitlab.com/ee/ci/caching/#sharing-caches-across-the-same-branch, it appears that 
- using `$CI_COMMIT_REF_NAME` actually allow us to share cache between all jobs of same branch/mr, which what we want 
- using `${CI_COMMIT_REF_SLUG}` is same but even cleaner
- using `$CI_COMMIT_REF_NAME:$CI_COMMIT_SHA` woud restrict cache to the commit, which seem too fine grained to be useful for caching
- using `"front-packages"` allow to share caches across different branche of same project, which can be even better

Reply:
After my initial comment I checked docs more in depth and found the following:
1. `${CI_COMMIT_REF_SLUG}` is docs documentation and as you point it is better than `${CI_COMMIT_REF_NAME}`
2. `$CI_COMMIT_REF_NAME:$CI_COMMIT_SHA` is more complex but in practice the same as `${CI_COMMIT_REF_SLUG}` cause as I see gitlab CI keeps a local cache per pipeline. To have a global cache among pipelines we need further configuration on infra side.
3. I think `"front-packages"` won' be ok in any case cause it will cause issues as we add/remove packages along the way

Reply:
++
Let's go with `    key: ${CI_COMMIT_REF_SLUG}` + add `-front` to it in prevision to also caching backend (composer) stuff and to avoid one from overwritting each other ([ex](https://docs.gitlab.com/ee/ci/caching/#examples))